lib/graphql_authorize/configuration.rb

```diff
@@ -7,6 +7,9 @@ class Configuration
     def initialize
       @auth_adapter = nil
       @auth_adapter_source = nil
+
+      I18n.load_path << Dir[File.expand_path("config/locales") + "/*.yml"]
+      I18n.default_locale = :en
```

Comment:
I'm not sure we need it here - there is a chance that it will override `default_locale` that was set up earlier. We can remove it completely (and move to specs) or just add `||=`.

Reply:
I'm not super sure about that, just want to make sure we're not overriding the value set by someone else (e.g. gem is required after the app initialised it's own `default_locale`)

Reply:
Oh wait, we seem to not need this line at all :) :en is already default https://github.com/svenfuchs/i18n/blob/master/README.md#ruby-without-rails

Reply:
Ah neat - I'll remove it 👍